README.md

```diff
@@ -306,13 +305,13 @@ func main() {
         log.Fatalf("error instantiating staking client: %s", err.Error())
     }
 
-    // Lists the rewards for the given address for the previous last 20 days, aggregated by day.
+     // Lists the rewards for the given address for May 1st 2024, aggregated by day.
```

Comment:
maybe a 2 day window at least ?

Reply:
I have a one day window in the TS client, hoping to keep things consistent here. Unless you feel it would also benefit a change there.

Reply:
Sure - then let's do 1 day for now.

Reply:
Switched it to 2 now 😅 . In the README it's only for 1 but in the example I made it for 2.